libindy_vdr/src/pool/runner.rs

```diff
@@ -217,16 +221,21 @@ impl PoolThread {
     }
 }
 
-async fn _perform_refresh(pool: &LocalPool, callback: Callback<RefreshResponse>) {
-    let result = perform_refresh(pool).await;
+async fn _perform_refresh(
+    pool: &LocalPool,
+    callback: Callback<RefreshResponse>,
+    cache: Option<Cache<String, (String, RequestResultMeta)>>,
+) {
+    let result = perform_refresh(pool, cache).await;
```

Comment:
We don't want to use the cache when performing a refresh, it needs to connect to one or more nodes to see if the pool state is up to date.

Reply:
Done